cli/integration_tests/task-dependencies/topological.t

```diff
@@ -24,3 +24,19 @@ Check my-app#build output
   Cached:    0 cached, 2 total
     Time:\s*[\.0-9]+m?s  (re)
   
+
+
+Graph
```

Comment:
What's the goal of this test? I'm not really that well versed in this part of the codebase.

Reply:
This is merged in from here: https://github.com/vercel/turbo/blob/main/cli/integration_tests/topological_deps/graph.t
The underlying config is:

```json
{
  "$schema": "https://turbo.build/schema.json",
  "pipeline": {
    "build": {
      "dependsOn": ["^build"]
    },
    "//#build": {
      "dependsOn": []
    }
  }
}

So I think the idea is to validate that the task graph is generated correctly